collection-scripts/gather

```diff
@@ -22,11 +22,14 @@ pre-install.sh ${BASE_COLLECTION_PATH}
 gather_namespaced_resources ${BASE_COLLECTION_PATH}
 gather_clusterscoped_resources ${BASE_COLLECTION_PATH}
 gather_noobaa_resources ${BASE_COLLECTION_PATH}
+gather_odf_client ${BASE_COLLECTION_PATH}
```

Comment:
`gather_odf_client` first called here.

Reply:
With condition, we are unable to get client CRs, pods, yamls when both provider and client are deployed in a single cluster. 
For cluster with only provider installed, the gather_odf_client will fail and only provider CRs are fetched